h3-quinn/Cargo.toml

```diff
@@ -19,6 +19,6 @@ quinn = { version = "0.10", default-features = false, features = [
     "futures-io",
 ] }
 quinn-proto = { version = "0.10", default-features = false }
-tokio-util = { version = "0.7.7" }
-futures = { version = "0.3.27" }
-tokio = { version = "1.28", features = ["io-util"], default-features = false }
+tokio-util = { version = "0.7.9" }
+futures = { version = "0.3.28" }
+tokio = { version = "1", features = ["io-util"], default-features = false }
```

Comment:
Is it true we work on 1.0? Usually we keep the minimum version that works. I suspect we use something that was newer than that. It's possible to check with `minimal-versions`.

Reply:
Hmm `cargo minimal-versions check --workspace` is successful on my machine. Is there something else to check?

Reply:
Alrighty, sounds good!